Formula/cedille.rb

```diff
@@ -0,0 +1,117 @@
+require "language/haskell"
+
+class Cedille < Formula
+  include Language::Haskell::Cabal
+
+  desc "Language based on the Calculus of Dependent Lambda Eliminations"
+  homepage "https://cedille.github.io/"
+
+  stable do
+    url "https://github.com/cedille/cedille/archive/v1.1.2.tar.gz"
+    sha256 "cf6578256105c7042b99a70a897c1ed60f856b28628b79205eb95730863b71cb"
+
+    resource "ial" do
+      url "https://github.com/cedille/ial/archive/v1.5.0.tar.gz"
+      sha256 "f003a785aba6743f4d76dcaafe3bc08bf879b2e1a7a198a4f192ced12b558f46"
+    end
+  end
+
+  head do
+    url "https://github.com/cedille/cedille.git"
+
+    resource "ial" do
+      url "https://github.com/cedille/ial.git"
+    end
+  end
+
+  depends_on "agda" => :build
+  depends_on "cabal-install" => :build
+  depends_on "ghc"
+
+  def install
+    resource("ial").stage buildpath/"ial"
+
+    cabal_sandbox do
+      # build tools
+      cabal_install_tools "alex", "happy", "cpphs"
+
+      # build dependencies
+      cabal_install "ieee754"
+
+      # use the sandbox when building with Agda
+      ENV["GHC_PACKAGE_PATH"] = "#{buildpath/Dir[".cabal-sandbox/*packages.conf.d/"].first}:"
+
+      # build
+      system "make", "core/cedille-core", "cedille-mac"
+    end
+
+    # binaries and elisp
+    bin.install "core/cedille-core"
+    mv "cedille-mac", bin/"cedille"
```

Comment:
```suggestion
    bin.install "cedille-mac" => "cedille"
```

Reply:
Ah, thanks, I meant to ask about this. I knew there'd be a way to do it properly.